docker/codex_scripts/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM continuumio/miniconda3
```

Comment:
Can you pin to a specific `miniconda3` version? (Probably whichever one is current and tagged `latest` at this time.)